src/back/TicketManager/docker-compose.yaml

```diff
@@ -23,6 +25,13 @@ services:
       - db
     volumes:
       - /var/ticket_manager/postgres:/var/lib/postgresql/data
+  payment-service:
+    container_name: payment-service
+    image: payment-service
+    build:
+      dockerfile: Dockerfile.payment
+    ports:
+      - "8081:80"
 networks:
```

Comment:
We could probably extend networks structure here so that `backend` and `PaymentService` communicate via enclosed channel not `localhost`